src/in-depth/signals-channels.rs

```diff
@@ -2,7 +2,7 @@ use std::time::Duration;
 use crossbeam_channel::{bounded, tick, Receiver, select};
 use anyhow::Result;
 
-fn ctrl_channel() -> Result<Receiver<()>, ctrlc::Error> {
+fn ctrl_channel() -> Result<Receiver<()>> {
```

Comment:
What was the motivation for switching this function from reporting `ctrlc::Error` to `anyhow::Error`?